filebeat/input/filestream/internal/input-logfile/manager.go

```diff
@@ -177,13 +177,15 @@ func (cim *InputManager) Create(config *conf.C) (v2.Input, error) {
 			" duplication, please add an ID and restart Filebeat")
 	}
 
+	cim.Logger.Infow("inputmanager ids", "ids", cim.ids)
 	cim.idsMux.Lock()
 	if _, exists := cim.ids[settings.ID]; exists {
-		cim.Logger.Errorf("filestream input with ID '%s' already exists, this "+
+		cim.Logger.Warnf("filestream input with ID '%s' already exists, this "+
 			"will lead to data duplication, please use a different ID", settings.ID)
 	}
 
 	cim.ids[settings.ID] = struct{}{}
+	cim.Logger.Infof("inputmanager added %q", settings.ID)
```

Comment:
we should not mentioned the input manager in INFO logs, it's internal implementation.

```suggestion
	cim.Logger.Infof("filestream with ID %q added", settings.ID)
```